### PR TITLE
Add 10.12.1-rc.2 to .drone.star

### PR DESCRIPTION
**Pull request by jnweiger**
(there was no rc1. we start with rc2, this time)

### CODE REVIEW

File: .drone.star [hunk 1]
```diff
@@ -1,5 +1,14 @@
 def main(ctx):
     versions = [
+        {
+            "value": "10.12.1-rc.2",
+            "qa": "https://download.owncloud.com/server/testing/owncloud-complete-20230404-qa.tar.bz2",
```
Comment (xoxys) [inline .drone.star:5]: Please remove the QA tarball. Daily QA bundle is used by default.

**Review by xoxys — CHANGES_REQUESTED** [review]

File: .drone.star [hunk 2]
```diff
@@ -1,5 +1,13 @@
 def main(ctx):
     versions = [
+        {
+            "value": "10.12.1-rc.2",
+            "tarball": "https://download.owncloud.com/server/testing/owncloud-complete-20230404.tar.bz2",
+            "tarball_sha": "94ec382e3c9f1fdc92bd5e022ea422d08a6e24537b6526f8e08dc8650c900475",
+            "php": "7.4",
+            "base": "v20.04",
+            "tags": ["10.12.1-rc.1"],
```
Comment (phil-davis) [inline .drone.star:9]: why does this say `rc.1` but "value" has `rc.2` ?

Comment (jnweiger) [reply .drone.star]: oops. Fixed.

File: .drone.star (hunk 1 above)
Comment (jnweiger) [reply .drone.star]: removed.

**Review by xoxys — APPROVED** [review]

**Review by phil-davis — APPROVED** [review]

**Pull request merged**